src/Processors/Transforms/Streaming/GlobalAggregatingTransform.h

```diff
@@ -28,6 +28,8 @@ class GlobalAggregatingTransform final : public AggregatingTransform
     bool prepareFinalization(Int64 min_watermark) override;
 
     void finalize(const ChunkContextPtr & chunk_ctx) override;
+
+    static constexpr VersionType IMPL_V2_MIN_VERSION = 3;
```

Comment:
lets just use VERSION_2 = 3;

Reply:
This naming is simple but confusing

Reply:
Client usually does do `if (v > v2 && v < v4) ... `